Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -2534,7 +2534,7 @@ def condition(self, fragment, stepList, key, hasHarvest):
 class UpgradeWorkflow_DDDDB(UpgradeWorkflow):
     def setup_(self, step, stepName, stepDict, k, properties):
         the_era = stepDict[step][k]['--era']
-        if 'Run3' in the_era  and '2023' not in the_era and 'Fast' not in the_era and "Pb" not in the_era:
+        if 'Run3' in the_era and '2023' not in the_era and '2024' not in the_era and 'Fast' not in the_era and "Pb" not in the_era:
```

Comment:
I am not totally sure to understand what's the intent here, but this line is necessary to avoid having runtime matrix failures. I am wondering if instead of excluding explicitly every "customization" on top of `Run3` one should just match exactly `Run3` and be done with it (but then I don't understand the purpose of this class).
Perhaps @cms-sw/pdmv-l2 or @cms-sw/upgrade-l2 can clarify.

Reply:
@srimanob do you have any idea of the usage of this `UpgradeWorkflow_DDDDB`. Sincerely I don't know/remember.

Reply:
This is to use DDD from DB, see 
https://github.com/cms-sw/cmssw/blob/master/Configuration/AlCa/python/autoCondModifiers.py#L34-L75